tests/congure_test/README.md

```diff
@@ -22,7 +22,7 @@ $ BOARD="<board>" make flash test
 It can also executed with pytest:
 
 ```console
-$ pytest tests/01-run.py
+$ PYTHONPATH=../../dist/pythonlibs pytest tests/01-run.py
```

Comment:
this needs `BOARD=foo` to work, maybe mention?

Reply:
It defaults to native, but will add!